modules/account-lib/src/coin/stx/transaction.ts

```diff
@@ -0,0 +1,171 @@
+import BigNumber from 'bignumber.js';
+import {
+  BufferReader,
+  PayloadType,
+  StacksTransaction,
+  TransactionSigner,
+  createStacksPrivateKey,
+  deserializeTransaction,
+  addressToString,
+  StacksMessageType,
+  createStacksPublicKey,
+  SingleSigSpendingCondition,
+  isSingleSig,
+  StacksPublicKey,
+  TransactionAuthField
+} from '@stacks/transactions';
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { SigningError } from '../baseCoin/errors';
+import { BaseKey } from '../baseCoin/iface';
+import { BaseTransaction, TransactionType } from '../baseCoin';
+import { SignatureData, TxData } from './iface';
+import { getTxSenderAddress, bufferToHexPrefixString, removeHexPrefix } from './utils';
+import { KeyPair } from './keyPair';
+
+export class Transaction extends BaseTransaction {
+  private _stxTransaction: StacksTransaction;
+  protected _type: TransactionType;
+
+  constructor(_coinConfig: Readonly<CoinConfig>) {
+    super(_coinConfig);
+  }
+
+  /** @inheritdoc */
+  canSign(key: BaseKey): boolean {
+    // TODO: implement this validation method
```

Comment:
What is the blocker here?

Reply:
stx transactions do not have source address to validate with the key provided.

Reply:
Ah ok - can we remove the TODO in that case?